Helper/ExtensionAttributeMerger.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * ClassyLlama_AvaTax
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @copyright  Copyright (c) 2018 Avalara, Inc.
+ * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+
+namespace ClassyLlama\AvaTax\Helper;
+
+use Magento\Framework\Api\ExtensibleDataInterface;
+use Magento\Framework\Api\ExtensionAttribute\JoinProcessorHelper;
+
+class ExtensionAttributeMerger
+{
+    /**
+     * @var \Magento\Framework\Api\ExtensionAttributesFactory
+     */
+    protected $extensionAttributesFactory;
+
+    /**
+     * @var JoinProcessorHelper
+     */
+    protected $joinProcessorHelper;
+
+    /**
+     * @param JoinProcessorHelper                               $joinProcessorHelper
+     * @param \Magento\Framework\Api\ExtensionAttributesFactory $extensionAttributesFactory
+     */
+    public function __construct(
+        JoinProcessorHelper $joinProcessorHelper,
+        \Magento\Framework\Api\ExtensionAttributesFactory $extensionAttributesFactory
+    )
+    {
+        $this->extensionAttributesFactory = $extensionAttributesFactory;
+        $this->joinProcessorHelper = $joinProcessorHelper;
+    }
+
+    protected function getExtensionAttributeMethodName($key)
```

Comment:
Add DocBlocks to this and the methods below. If you copied this code from core methods, add a `@see` reference